### PR TITLE
docs: various doc fixups

### PR DESCRIPTION
**Pull request by ldelossa**
Signed-off-by: ldelossa <ldelossa@redhat.com>

### CODE REVIEW

**Review by jan-zmeskal — APPROVED** [review]
Looks good. Thank you for making the effort to make the code easy to understand!

File: internal/vulnstore/updater.go [hunk 1]
```diff
@@ -31,6 +31,10 @@ type Updater interface {
 	// updater.
 	GetLatestUpdateRef(context.Context) (uuid.UUID, error)
 	// DeleteUpdateOperations removes an UpdateOperation.
+	// A call to GC must be ran after this to garbage collect vulnerabilities associated
```
Comment (hdonnay) [inline internal/vulnstore/updater.go:34]:
```suggestion
	// A call to GC must be run after this to garbage collect vulnerabilities associated
```

Pretty sure past-tense is incorrect there.

File: docs/getting_started/libvuln_usage.md [hunk 2]
```diff
@@ -10,15 +10,15 @@ type Opts struct {
...
+	// A connection string to the database Lbvuln will use.
 	ConnString string
 	// An interval on which Libvuln will check for new security database
 	// updates.
 	//
 	// This duration will have jitter added to it, to help with smearing load on
 	// installations.
 	UpdateInterval time.Duration
-	// Determines if Libvuln will manage database migrations
+	// Determines if Livuln will manage database migrations
```
Comment (hdonnay) [inline docs/getting_started/libvuln_usage.md:21]: Introduces a typo.

File: docs/getting_started/libvuln_usage.md [hunk 3]
```diff
@@ -10,15 +10,15 @@ type Opts struct {
 	// The maximum number of database connections in the
 	// connection pool.
 	MaxConnPool int32
-	// A connection string to the database Libvuln will use.
+	// A connection string to the database Lbvuln will use.
```
Comment (hdonnay) [inline docs/getting_started/libvuln_usage.md]: ???

File: libvuln/libvuln.go [hunk 4]
```diff
@@ -95,8 +95,11 @@ func (l *Libvuln) UpdateOperations(ctx context.Context, updaters ...string) (map
 	return l.store.GetUpdateOperations(ctx, updaters...)
 }
 
-// DeleteUpdateOperations removes one or more update operations and their
-// associated vulnerabilities from the vulnerability database.
+// DeleteUpdateOperations removes an UpdateOperation.
```
Comment (hdonnay) [inline libvuln/libvuln.go:98]:
```suggestion
// DeleteUpdateOperations removes UpdateOperations.
```

File: libvuln/libvuln.go [hunk 5]
```diff
@@ -95,8 +95,11 @@ func (l *Libvuln) UpdateOperations(ctx context.Context, updaters ...string) (map
...
+// A call to GC or GCFull must be ran after this to garbage collect vulnerabilities associated
```
Comment (hdonnay) [inline libvuln/libvuln.go:99]:
```suggestion
// A call to GC or GCFull must be run after this to garbage collect vulnerabilities associated
```

File: libvuln/libvuln.go [hunk 6]
```diff
@@ -124,9 +127,9 @@ func (l *Libvuln) LatestUpdateOperation(ctx context.Context) (uuid.UUID, error)
 }
 
 // GC will cleanup any update operations older then the configured UpdatesRetention value.
-// GC returns
-//
 // GC is throttled and ensure its a good citizen to the database.
+//
+// the returned int the number of of outstanding UpdateOperations not deleted due to throttling.
```
Comment (hdonnay) [inline libvuln/libvuln.go:132]:
```suggestion
// The returned int is the number of outstanding UpdateOperations not deleted due to throttling.
```

File: libvuln/libvuln.go [hunk 7]
```diff
@@ -139,9 +142,8 @@ func (l *Libvuln) GC(ctx context.Context) (int64, error) {
 // and stale vulnerabilites are removed in accordance with the UpdateRetention
 // value.
 //
-// GCFull may return an error accompanied by its other two return values;
-// the number of oustanding update operations to remove and the number of
-// vulnerabilities deleted.
+// GCFull may return an error accompanied by its other return value;
```
Comment (hdonnay) [inline libvuln/libvuln.go:145]:
```suggestion
// GCFull may return an error accompanied by its other return value,
```

The second clause is not a full sentence, so a semicolon is not appropriate.

**Review by hdonnay — CHANGES_REQUESTED** [review]

**Review by hdonnay — APPROVED** [review]

**Pull request merged**